src/types/react-table-config.ts

```diff
@@ -35,10 +35,10 @@ import {
   UsePaginationInstanceProps,
   UsePaginationOptions,
   UsePaginationState,
-  // UseResizeColumnsColumnOptions,
+  UseResizeColumnsColumnOptions,
   UseResizeColumnsColumnProps,
-  // UseResizeColumnsOptions,
-  UseResizeColumnsState,
+  UseResizeColumnsOptions,
+  // UseResizeColumnsState,
```

Comment:
sure not needed?

Reply:
I am pretty sure about this:
![image](https://user-images.githubusercontent.com/36186912/144609257-1f0d4c33-9360-4c04-9270-97873756b245.png)